application/frontend/src/pages/Docs/docs.scss

```
@@ -33,25 +39,199 @@
+    background: var(--secondary);
+    border: 1px solid var(--border);
+    border-radius: var(--radius);
     overflow: hidden;
-    box-shadow: 0 10px 30px rgba(0, 0, 0, 0.25);
+    color: var(--foreground);
 
     .swagger-ui {
-      border: none;
+      background: transparent;
+      color: var(--foreground);
+      font-family: 'Inter', system-ui, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)